appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/api/visitor/ApiContext.java

```diff
@@ -64,4 +64,10 @@
      * The created operation currently being worked on.
      */
     Operation getWorkingOperation();
+
+    /**
+     * @param type any class, not null
+     * @return true, if the give type is a known type in this context, else false
+     */
+    boolean isApplicationType(Class<?> type);
```

Comment:
I added this to allow the `ApplicationProcessor` logic to make this check even though the information on this is within the `OpenApiWalker` that creates the `ApiContext`.